### ISSUE TITLE
32 Players in scope RedM limitation

### ISSUE THREAD

**Issue by d22tny** [citizenfx/fivem#2932]
### What happened?

First off, I just want to say thank you for everything you’ve done so far. You’re an amazing team, and every time I’ve reported an issue, you’ve managed to fix it in no time—sometimes even within a day. That level of dedication is rare, and I’ve always appreciated it.

That’s why I feel confident coming to you with this—not as a bug report, but as a major issue that’s really holding RedM back, especially for roleplay servers. It’s the “32 players in scope” limitation.

Why is this such a big deal? Because RedM (like FiveM) is all about roleplaying. Honestly, I’d say 90% of the servers are roleplay-focused. And roleplay is all about people—interacting with them, running into them naturally, creating those fun, unexpected moments that make you want to log back in every day.

But the 32-player scope limit kind of kills that vibe. It’s frustrating to be in a town or busy area, knowing there are a bunch of other players around, but you can’t see or interact with half of them. It’s like the world gets weirdly “cut off,” and it ruins the immersion.

It’s even worse when you think about how massive Red Dead’s map is. The world is built for sprawling adventures and big communities, but the scope issue makes it hard to really enjoy that. It feels like we’re stuck in a smaller space than we should be, which is such a shame because this game is perfect for creating big, lively, and dynamic environments.

I just created this as a reminder.. to try and make you remember that this is a serious issue,

The thing that made me decide to open this issue happened just last night. A gang on the server organized a scene at the Saint Denis theater—a short performance that lasted about 30 minutes. It was amazing! But we had to spend two days beforehand planning everything: making sure staff didn’t go near the area, carefully picking 2-3 people from other gangs, and even factoring in the possibility of random players showing up, all just to stay within the 32-player limit. It was kind of sad, knowing there were hundreds of players on the server at the time, but only 32 could actually enjoy this special moment.

![image](https://github.com/user-attachments/assets/9360ceb4-a6f8-448c-acee-0921400eca44)

Just imagine the immersion that there could be idk.. at least 64 players ?

### Expected result

64, maybe 128 players in scope

### Reproduction steps

-

### Importancy

Slight inconvenience

### Area(s)

RedM

### Specific version(s)

RedM ( any version, any server build )

### Additional information

Peace!

**Comment by Gltch3d** [comment]
Sync in general with RedM is very disappointing such as peds being killed out of scope appearing to be in some sort of "barely alive" state when a player goes into scope. Phantom wagons becoming completely stuck requiring scripts to detect and delete them.. just to name a few. But as you stated the scope limitation is very frustrating. Correct me if I am wrong but isnt this a card coded limitation within the game itself? It would be nice to see positive changes being made in support of RedM rather than having minimal updates as a by product of changes being made for FXServer/FiveM mainly.

**Comment by outsider31000** [comment]
Just some *unofficial* clarification, unfortunately there's no one working on RedM specifically from Cfx. 
R* is focused on FiveM at the moment although I doubt that they will change since they stopped RDO updates as we all know.
Things that get added is because they are added to FiveM and works for both ways.
In other words, is up to contributors now as it always have been.

We are all aware of the issues but without contributions I'm afraid the player scope increase won't see the light of the day any time soon.

**Comment by d22tny** [comment]
Yes, i know, that's why i opened this, to remind the greatness of RedM, and maybe who knows, it's something similar to FiveM ? Maybe it's worth checking ?

**Comment by Cephy314** [comment]
> Yes, i know, that's why i opened this, to remind the greatness of RedM, and maybe who knows, it's something similar to FiveM ? Maybe it's worth checking ?

Unfortunately, I beleive they already solved this for FiveM but the fix does not work for RedM, so unless we get a talented dev able to focus on this problem and sort it out or direct assistance from R* I am going to assume were SOL until rdr4

**Comment by teddyyGO** [comment]
This issue is still active
